templates/birds/macros.j2

```diff
@@ -1,58 +1,55 @@
 {% import "utility.macros.j2" as utility %}
 
 
-{% macro section(bird, oss, posts=none) %}
-  <div data-module='bird' class="h-card" id="bird-{{ bird.author }}">
-    {{ utility.icon(bird.author, size='face') }}
+{% macro section(page, oss=none, posts=none) %}
+  {% set author = page.config.bird %}
+
+  <div data-module='bird' class="h-card" id="bird-{{ author }}">
+    {{ utility.icon(author, size='face') }}
 
     <header>
-      {{ title(bird.name, bird.author, bird.no_posts) }}
-      {{ link_list(bird.links[0]) }}
+      {{ title(page.title, author) }}
```

Comment:
Because we're no longer passing in a `no_posts` boolean, this creates a broken link for birds without posts (currently David).